### PR TITLE
Update devices.js

### PR DESCRIPTION
**Pull request by supergroover**
this add the Gledopto GL-C-009S device

### CODE REVIEW

**Comment by Koenkk** [comment]
Note to self, after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/556

File: devices.js [hunk 1]
```diff
@@ -6692,6 +6692,19 @@ const devices = [
         description: 'Zigbee LED controller dimmer',
         extend: preset.gledopto.light_onoff_brightness,
     },
+    {
+        zigbeeModel: ['GL-C-009S'],
+        fingerprint: [
```
Comment (Koenkk) [inline devices.js:6697]: I don't expect this `fingerprint` to be needed (`zigbeeModel`) is enough, can you try removing it

**Review by supergroover — COMMENTED** [review]
Add GL-C-009S, remove fingerpint

**Comment by Koenkk** [comment]
Looks something went wrong, lot of unexpected changes under: https://github.com/Koenkk/zigbee-herdsman-converters/pull/2059/files

**Pull request closed without merging**